tests/test_regex_identifier.py

```diff
@@ -73,13 +109,37 @@ def test_ctf_flag_uppercase():
 def test_ethereum():
     r = regex_identifier.RegexIdentifier()
     res = r.check(["0x52908400098527886E0F7030069857D2E4169EE7"])
-    assert "Ethereum Wallet" in res[0]["Regex Pattern"]["Name"]
+    assert "Ethereum (ETH) Wallet Address" in res[0]["Regex Pattern"]["Name"]
 
 
 def test_bitcoin():
     r = regex_identifier.RegexIdentifier()
     res = r.check(["1KFHE7w8BhaENAswwryaoccDb6qcT6DbYY"])
-    assert "Bitcoin" in res[0]["Regex Pattern"]["Name"]
+    assert "Bitcoin (BTC) Wallet Address" in res[0]["Regex Pattern"]["Name"]
+
+
+def test_monero():
+    r = regex_identifier.RegexIdentifier()
+    res = r.check(["47DF8D9NwtmefhFUghynYRMqrexiZTsm48T1hhi2jZcbfcwoPbkhMrrED6zqJRfeYpXFfdaqAT3jnBEwoMwCx6BYDJ1W3ub"])
+    assert "Monero (XMR) Wallet Address" in res[0]["Regex Pattern"]["Name"]
+
+
+def test_litecoin():
+    r = regex_identifier.RegexIdentifier()
+    res = r.check(["LRX8rSPVjifTxoLeoJtLf2JYdJFTQFcE7m"])
+    assert "Litecoin (LTC) Wallet Address" in res[0]["Regex Pattern"]["Name"]
+
+
+def test_bitcoincash():
+    r = regex_identifier.RegexIdentifier()
+    res = r.check(["bitcoincash:qzlg6uvceehgzgtz6phmvy8gtdqyt6vf359at4n3lq"])
+    assert "Bitcoin Cash (BCH) Wallet Address" in res[0]["Regex Pattern"]["Name"]
+
+
+def test_ripple():
+    r = regex_identifier.RegexIdentifier()
+    res = r.check(["rBPAQmwMrt7FDDPNyjwFgwSqbWZPf6SLkk"])
+    assert "Ripple (XRP) Wallet Address" in res[0]["Regex Pattern"]["Name"]
```

Comment:
This is annoying, but can you also add all the tests to:
https://github.com/bee-san/pyWhat/blob/main/fixtures/file

And copy / paste into:
https://github.com/bee-san/pyWhat/blob/main/tests/test_click.py

Because of how many regexes this has, we need to be on the ball with testing. We need to have **a lot** of tests to make sure we don't slip up :-)